_posts/2022-01-20-reached-4000-users.md

```diff
@@ -0,0 +1,21 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Usegalaxy.eu surpassed 40.000 registered users'
+tags: [galaxy, support, society, report]
+supporters:
+- galaxy-europe
+---
+
+We are of celebration! The last November usegalaxy.eu reached 40.000 registered users! Over the last year, the number of registered users has almost doubled, from 25,000 to more than 41,000 by January 2022. As we can see in figure 1, the growth in the number of new registrants shows an almost exponential trend since 2013.
+    
+![Total number of registered](/assets/media/reached_40000/total_users_galaxyeu.png)
```

Comment:
can we skip the part 2013-2019 ? Galaxy was not open at this time, or I guess we need to mention this.

From 2013-2019 it was only open for Freiburg researchers. So usually we count from 2019.

Reply:
Ah ok; as you consider better.